hydrocarbon_problem/agents/logger.py

```diff
@@ -0,0 +1,57 @@
+from acme.utils.loggers import Logger
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+class ListLogger(Logger):
+    """Manually save the data to the class in a dict. Currently only supports scalar history
+    inputs."""
+    def __init__(self):
```

Comment:
Can this logger also be used to store the returns and column data? So that afterwards we can see what the column specs are of the best column

Reply:
Yes - we could add a `get_info` method to the environment, and call it on each step, and then log the information. I'll make an initial version of this and then you could add an extra info you are interested in?